perfmetrics/scripts/fio/fio_metrics.py

```diff
@@ -43,12 +44,130 @@
 P50 = '50.000000'
 P90 = '90.000000'
 P95 = '95.000000'
-LAT_20_PERC = 'lat_20_perc'
-LAT_50_PERC = 'lat_50_perc'
-LAT_90_PERC = 'lat_90_perc'
-LAT_95_PERC = 'lat_95_perc'
 IO_BYTES = 'io_bytes'
 
+NAME = 'name'
+JSON_NAME = 'json_name'
+FORMAT = 'format'
+DEFAULT = 'default'
+LEVELS = 'levels'
+CONVERSION = 'conversion'
+NS_TO_S = 10**(-9)
+
+""" REQ_JOB_PARAMS:
+NAME: key for the parameter, to be used when creating parameter dict for each
+Job
+JSON_NAME: Key for parameter inside 'global options'/'job options' dictionary
```

Comment:
its not clearly getting conveyed that we can use any value for NAME but JSON_NAME has to match the fio job spec. please update the comment

Reply:
done